app/components/ResultsFilter/elements/PerPageDropdown/PerPageDropdown.js

```diff
@@ -3,18 +3,15 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import LabelText from '@govuk-react/label-text'
 import { SelectInput } from '@govuk-react/select'
-import { SPACING } from '@govuk-react/constants'
 
 const DropdownContainer = styled.div`
-  margin-left: ${SPACING.SCALE_3};
-
   span {
     font-weight: 700;
   }
 `
 
 const PerPageDropdown = ({ handleChange, totalResults, perPage }) => {
-  const defaultValue = totalResults <= 10 ? totalResults : perPage
+  const defaultValue = totalResults <= 20 && perPage !== 10 ? totalResults : perPage
```

Comment:
This was confusing to start with, mainly due to the magic numbers.  But I think I've got it, 20 is new default per page count. However, that is the second option in the list of [10,20,50,100].  We want the perPage count to be dynamic but only if 10 isn't selected and because it's less than the default. 

Not sure how to make this clearer, maybe if the 20 was stuffed into a  defaultPerPage variable? I had to look through the other parts of the code to figure put 20 is now the default, whereas before it was 10.

What do you think?